.github/workflows/master-e2e.yaml

```diff
@@ -149,12 +149,28 @@ jobs:
       RANCHER_VERSION: ${{ inputs.rancher_version }}
       TIMEOUT_SCALE: 3
     steps:
+      # Build rancher-turtles nightly chart
+      - name: Check out rancher-turtles repo
+        uses: actions/checkout@v3
+        with:
+          repository: rancher-sandbox/rancher-turtles
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: go.mod
+      - name: Make chart
+        run: make e2e-image
+      - name: Copy chart file
+        run: sudo cp ${{ github.workspace }}/out/package/rancher-turtles-0.0.1.tgz ${{ runner.temp }}
+
       - name: Checkout
         uses: actions/checkout@v3
       - name: Install Go
         uses: actions/setup-go@v3
         with:
           go-version-file: tests/go.mod
+      - name: Copy chart file for chartmuseum
+        run: sudo cp ${{ runner.temp }}/rancher-turtles-0.0.1.tgz ${{ github.workspace }}/tests/assets
```

Comment:
Not happy about using a fixed version at two places (see line 164).

Reply:
Its static tag being set from here, https://github.com/rancher-sandbox/rancher-turtles/blob/main/Makefile#L507
Agreed, needs a dynamic tag

Reply:
I guess the version does not matter here but we need to specify one when the chart is built. It's a one-shot chart only to deploy the latest dev version. However, it may have to be improved when we talk about upgrade.